_data/simple-icons.json

```diff
@@ -7708,6 +7708,11 @@
             "source": "https://www.jetbrains.com/company/brand/logos/",
             "guidelines": "https://www.jetbrains.com/company/brand/"
         },
+        {
+            "title": "Rimac",
+            "hex": "FFFFFF",
```

Comment:
Why did you go with `#FFFFFF`? From the website I would maybe expect a color like `#0A222E` :thinking:

Reply:
Hi. I assumed the color would follow as the logo in the main page, therefore I put it as #FFFFFF. I will amend this to #0A222E.